src/test/java/uk/gov/hmcts/reform/hmc/validator/BeanValidatorTest.java

```diff
@@ -507,4 +508,37 @@ void whenInvalidPartyRoleIsEmpty() {
         assertTrue(validationErrors.contains(PARTY_ROLE_EMPTY));
     }
 
+    @Test
+    void whenValidPartyRoleIs40Characters() {
+        PartyDetails partyDetails = new PartyDetails();
+        partyDetails.setPartyID("XXX1");
+        partyDetails.setPartyType(PartyType.IND.getLabel());
+        partyDetails.setPartyRole("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMN");
+        Set<ConstraintViolation<PartyDetails>> violations = validator.validate(partyDetails);
+        assertTrue(violations.isEmpty());
+        List<String> validationErrors = new ArrayList<>();
```

Comment:
Don't need lines 519 - 524? Finish tests once we assert violations is empty

Reply:
done

Reply:
Updated in 96433765a84f6a5fe58bb8429db9c033c06eeaeb